### ISSUE TITLE
Can no longer trade with guests or Traders

### ISSUE THREAD

**Issue by MaXX2341** [OrionFive/Hospitality#94]
At the first time a guest arrived at my village i could trade with them. The second time a trader visited me and i traded with him, but when i confirmed the purchase of his items and tried again to speak with him for traiding it no longer worked. Also when the next guest arrived, i could not traid with him.

https://gist.github.com/125ee2221e03f2d7cb9512b99722fa1c

https://gist.github.com/8d30ee41282534d0b7baa3ed6f1b0904

**Comment by OrionFive** [comment]
You've got console spam. This means the logs are pretty useless. You will have to fix that first. 
Edit your save game and find the object with the ID MinifiedFurniture505690. Remove the whole object, but be careful you don't break the structure. So make a backup of the file. 

Once that's fixed, try to trade again and then send me the log.

**Comment by MaXX2341** [comment]
Hey, I found the problem! The problem was, that i tried to reinstall a guest bed at a new position, but my colonist picked the guest bed up and placed a normal bed, which I switched in guest-mode. Now i had a unusable guest bed item at the ground. What i forgott was that some time later the bed item disappears and i could no longer traid with traders. Which means that the MinifiedFurniture505690 was the invisible guest bed item at the ground which the game tried to draw. 
Thank you for your help :D
